dogfood/main.tf

```diff
@@ -113,6 +69,47 @@ data "coder_git_auth" "github" {
 
 data "coder_workspace" "me" {}
 
+module "dotfiles" {
+  source   = "https://registry.coder.com/modules/dotfiles"
+  agent_id = coder_agent.dev.id
+}
+
+module "git-clone" {
+  source   = "https://registry.coder.com/modules/git-clone"
+  agent_id = coder_agent.dev.id
+  url      = "https://github.com/coder/coder"
+  path     = data.coder_parameter.repo_dir.value
+}
+
+module "personalize" {
+  source   = "https://registry.coder.com/modules/personalize"
+  agent_id = coder_agent.dev.id
+}
+
+module "code-server" {
+  source   = "https://registry.coder.com/modules/code-server"
+  agent_id = coder_agent.dev.id
+}
+
+module "jetbrains_gateway" {
+  source            = "https://registry.coder.com/modules/jetbrains-gateway"
+  agent_id          = coder_agent.dev.id
+  agent_name        = "dev"
```

Comment:
Why do we need both id and name? Maybe the name can't be inferred from the linked resource?

Reply:
Yes. The gateway URL requires this.